Tasks/PackerBuildV0/task.json

```diff
@@ -56,6 +56,22 @@
             },
             "defaultValue": "builtin"
         },
+        {
+            "name": "isManagedImage",
+            "type": "boolean",
+            "label": "Managed VM disk image",
+            "required": true,
+            "helpMarkDown": "Check if Managed disk image is to be created for auto generated Packer template or if Managed disk image is generated by user provisioned Packer template.",
```

Comment:
isn't the managed disk support only for auto-generated packer templates?

Reply:
In my opinion it will be for both. In case of user provisioned templates user would need to provided correct managed disk image creation template. The task will parse the variables depending on whether managed disk image is created or not.

**For Unmanaged Disk packer command Output has following tokens**
* export var PackerLogTokenImageUri = "**OSDiskUri**";
* export var PackerLogTokenStorageLocation = "**StorageAccountLocation**";

**and for managed disk packer output will have following tokens**
* export var PackerLogTokenManagedResourceGroupName = "**ManagedImageResourceGroupName**";
* export var PackerLogTokenManagedImageName = "**ManagedImageName**";
* export var PackerLogTokenManagedImageLocation = "**ManagedImageLocation**";

Reply:
Got it. Would it be better to keep help-text simple? Can we just say "check if generated image is a managed image"

Reply:
Will do.

Reply:
Done.